apps/site/assets/ts/projects/components/MoreProjectsTable.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { ReactElement } from "react";
+import MoreProjectsRow from "./MoreProjectsRow";
+import { Project } from "./Project";
+
+interface Props {
+  projects: Project[];
+}
+
+const MoreProjectsTable = ({ projects }: Props): ReactElement<HTMLElement> => (
+  <table className="c-more-projects-table" aria-label="More Projects">
+    <colgroup>
```

Comment:
I think you can remove this colgroup element.

The thead with tr & th for the table headers is all you need.

A colgroup is for a header that spans more than one column. Kind of an odd table situation that you don't see often, but if you had three columns that applied to one thing and then three columns to the right that applied to another thing, you'd group 3 and 3 in their own  colgroups.